### PR TITLE
Added docs how to use a private package index

### PR DESCRIPTION
**Pull request by mofe23**
#192

### CODE REVIEW

File: docs/how-to/use_private_package_index.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Using a private package index
+=============================
+
+
+If your project depends on packages not available on pipy, e.g. if you use briefcase in a corporate environment and
+your unfortunately not allowed to open source your packages, you need to set up a private package index and configure
```
Comment (phildini) [inline docs/how-to/use_private_package_index.rst]: you're

File: docs/how-to/use_private_package_index.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+`pip` to use it to enable briefcase to find your packages.
+
+The fastest way to get a local index up and running on unix or osx is to use _pypiserver which we will use in
+this howto, a more advanced options could be _devpi.
```
Comment (phildini) [inline docs/how-to/use_private_package_index.rst]: Cut the bit about devpi, not necessary.

File: docs/how-to/use_private_package_index.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+.. _pypiserver pipyserver: https://pypi.org/project/pypiserver/
+.. _devpi devpi: https://pypi.org/project/devpi/
+
+
+1. Install a local pypiserver
+-----------------------------
+In case your already running your own index, skip this step.::
```
Comment (phildini) [inline docs/how-to/use_private_package_index.rst]: you're

File: docs/how-to/use_private_package_index.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    password: my_password
+
+
+3. Push your sdist
+------------------
+
+Using setuptools (legacy)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not you should be able to push an sdist to your local index using::
```
Comment (phildini) [inline docs/how-to/use_private_package_index.rst]: Not -> Now?

File: docs/how-to/use_private_package_index.rst [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+    $ python setup.py sdist upload -r local
+
+
+This way of publishing packages is deprecated, still works with `pypi-server` but will fail if you use `devpi`.
```
Comment (phildini) [inline docs/how-to/use_private_package_index.rst]: Remove reference to devpi

File: docs/how-to/use_private_package_index.rst [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+Using twine (recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you set up a `devpi` server instead of `pipy-server` you have to use twine.
+
+When you push the package for the first time, you need to register it:::
+
+    twine register -r local dist/*.tar.gz
+
+now your able to upload it:::
```
Comment (phildini) [inline docs/how-to/use_private_package_index.rst]: you're

**Comment by mofe23** [comment]
Thx for reviewing my typos, you're the best :)
my english teacher would have killed me. . .

**Pull request merged**